Modules/DataModel/Sources/PocketCastsDataModel/Private/Managers/EndOfYearDataManager.swift

```diff
@@ -136,6 +136,7 @@ class EndOfYearDataManager {
                 let query = """
                             SELECT DISTINCT \(DataManager.episodeTableName).uuid,
                                 COUNT(DISTINCT podcastUuid) as numberOfPodcasts,
+                                COUNT(DISTINCT \(DataManager.episodeTableName).uuid) as numberOfEpisodes,
```

Comment:
@ashiagr FYI: for the top categories we now have to calculate the number of episodes.